### PR TITLE
Add a few references to the FE_Q_iso_Q1 element documentation.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

File: doc/doxygen/references.bib [hunk 1]
```diff
@@ -1478,3 +1478,40 @@ @article{kronbichler2019multigrid
...
+  publisher = {Springer Science and Business Media {LLC}},
+  volume = {52},
+  number = {2},
+  pages = {383--400},
+  author = {D. Boffi and N. Cavallini and F. Gardini and L. Gastaldi},
+  title = {Local Mass Conservation of Stokes Finite Elements},
+  journal = {Journal of Scientific Computing}
+}
+
+@article{taho73,
```
Comment (kronbichler) [inline doc/doxygen/references.bib:1496]: Can we rename this into 
```suggestion
@article{Taylor73,
```
to give those people who do not want to click the reference an idea what the paper is about? (I do not like to click on reference links and just look at the name of the web page linked to, which picks up this reference and thus I like expressive names for the papers.)

**Review by kronbichler — COMMENTED** [review]
Looks good, thank you for the references. I added just a small comment below.

**Comment by bangerth** [comment]
So changed! :-)

**Review by kronbichler — APPROVED** [review]

**Comment by kronbichler** [comment]
Thanks!

**Comment by bangerth** [comment]
I tend to think that we ought to have more references to the literature, and this one was a fun find while working on a project in which the Taylor-Hood element was important.

**Pull request merged**